### PR TITLE
PR for Add notes

### PR DESCRIPTION
**Pull request by manismk**
- Added add note functionality
- Read and store data in firebase 
- [Link](https://deploy-preview-6--key-notes.netlify.app/login)

### CODE REVIEW

File: src/components/notes-form/NotesForm.jsx [hunk 1]
```diff
@@ -51,22 +52,25 @@ export const NotesForm = ({ closeForm }) => {
...
               onClick={() => {
-                notesDispatch({
-                  type: notesAction.ADD_NOTES,
-                  payload: {
-                    title: notesData.title,
-                    notes: notesData.enteredNotes,
-                  },
-                });
-                closeForm();
+                const { error } = handleNotesValidation(
```
Comment (shraddha-amber) [inline src/components/notes-form/NotesForm.jsx]: move this logic part into a separate handler function

File: src/pages/Profile/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          <h1 className="heading--3 text--center">My Account</h1>
+          <div className="avatar--container">
+            <img
+              src="https://randomuser.me/api/portraits/men/41.jpg"
+              alt="Randomuser"
+              className="avatar avatar--circle avatar--xl"
+            />
+          </div>
+
+          {user && user ? (
```
Comment (shraddha-amber) [inline src/pages/Profile/Profile.jsx]: i think this should work by ternary, no need to use user &&

File: src/utils/handleNotesValidation.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+export const handleNotesValidation = (title, note) => {
+  let error = "";
+  if (title.length === 0 && note.length === 0)
```
Comment (shraddha-amber) [inline src/utils/handleNotesValidation.js:2]: instead of multiple if statements, write if-else ladder, or directly return the error from the same if block

**Review by shraddha-amber — CHANGES_REQUESTED** [review]

File: src/components/notes-form/NotesForm.jsx (hunk 1 above)
Comment (manismk) [reply src/components/notes-form/NotesForm.jsx]: Thanks @shraddha-1402  I have made the changes

File: src/pages/Profile/Profile.jsx (hunk 2 above)
Comment (manismk) [reply src/pages/Profile/Profile.jsx]: Done

File: src/utils/handleNotesValidation.js (hunk 3 above)
Comment (manismk) [reply src/utils/handleNotesValidation.js:2]: I have made early return on all the if blocks

**Review by shraddha-amber — APPROVED** [review]

**Review by Pratik1005 — APPROVED** [review]
Good folder structure and nice UI

**Pull request merged**